Jumpscale/clients/blockchain/tfchain/TFChainCapacity.py

```diff
@@ -76,6 +144,8 @@ def reserve_s3(self, email, threebot_id, location, size=1, source=None, refund=N
                     - 1 => 500 GiB of storage
                     - 2 => 1000 GiB of storage
         :type size: int, optional
+        :param duration: number of months the reservation should be valid for
```

Comment:
I wonder if the minimal granularity of a reservation should be a month?
@andhartl @despiegk any comment on that ?